agent/instrumentation/azure-functions/src/main/java/io/opentelemetry/javaagent/instrumentation/azurefunctions/InvocationInstrumentation.java

```diff
@@ -57,7 +64,20 @@ public static Scope methodEnter(@Advice.Argument(0) Object request)
               .extract(Context.root(), traceContext, GETTER);
       SpanContext spanContext = Span.fromContext(extractedContext).getSpanContext();
 
-      return Context.current().with(Span.wrap(spanContext)).makeCurrent();
+      String invocationId =
```

Comment:
Lines 67 to 79  could be combined into one method (construction of an `AzureFunctionsCustomDimensions` object from a request and a trace context). Maybe something in `AzureFunctionsCustomDimensions` class?